### PR TITLE
OB console tgui

### PR DESCRIPTION
**Pull request by stanalbatross**
## About The Pull Request

image
image
(second warhead fixed)

## Why It's Good For The Game

less awful html ui

## Changelog

:cl:
ui: converted the OB console to tgui
/:cl:

### CODE REVIEW

**Review by morrowwolf — CHANGES_REQUESTED** [review]
operable check

**Comment by stanalbatross** [comment]
> operable check

was in it

File: tgui/packages/tgui/interfaces/OrbitalCannonConsole.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+              <Icon name="exclamation-triangle" />
+              {' No cannon linked to console!'}
+            </Dimmer>
+          )}
+          {!data.linkedtray && (
+            <Dimmer fontSize="32px">
+              <Icon name="exclamation-triangle" />
+              {' No tray linked to console!'}
+            </Dimmer>
+          )}
```
Comment (spookydonut) [inline tgui/packages/tgui/interfaces/OrbitalCannonConsole.js:145]: just change the text, no need to have the same block 3 times

**Review by spookydonut — CHANGES_REQUESTED** [review]

File: tgui/packages/tgui/interfaces/OrbitalCannonConsole.js (hunk 1 above)
Comment (stanalbatross) [reply tgui/packages/tgui/interfaces/OrbitalCannonConsole.js]: done

**Review by spookydonut — APPROVED** [review]

**Pull request merged**